src/components/common/Alpha.js

```diff
@@ -5,10 +5,24 @@ import * as alpha from '../../helpers/alpha'
 import Checkboard from './Checkboard'
 
 export class Alpha extends (PureComponent || Component) {
+  constructor(props) {
+    super(props)
+
+    this.container = null
+  }
+  
   componentWillUnmount() {
     this.unbindEventListeners()
   }
 
+  componentDidMount() {
+    // e.preventDefault() does not work properly on React synthetic touch event.
+    // As a workaround, touch events have to be added to directly to node
+    // https://github.com/facebook/react/issues/9809#issuecomment-414072263
+    this.container.addEventListener('touchstart', this.handleChange)
+    this.container.addEventListener('touchmove', this.handleChange)
```

Comment:
It should be `removeEventListener`-ed inside `componentWillUnmount` or `unbindEventListeners`.

Reply:
@fsubal It's done. Thanks for reviewing :smile:

Aren't they GCed when unmounted as they are attached to the node in the component. But I guess it is safe to remove them explicitly in `componentWillUnmount`.